sublime_music/ui/icons/star-half-empty.svg

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What is the point of this file?

Reply:
This has been removed. I guess it was part of the original branch to support ratings in steps of 0.5. This is not supported by subsonic, so I have removed it.